packages/grafana-ui/src/saga-themes/code/themes/dark.ts

```diff
@@ -0,0 +1,67 @@
+export default {
+  color: {
+    background: {
+      ui: {
+        canvas: 'hsl(220 11.8% 7.2%)',
+        primary: 'hsl(220 11.8% 10.8%)',
+        secondary: 'hsl(220 11.8% 15.6%)',
+        tertiary: 'hsl(220 11.8% 28.8%)',
+        overlay: 'hsl(220 11.8% 10.8% / 0.4)',
```

Comment:
having all colors use hsl syntax makes it a bit hard to reason about them without using an editor (or compare them), 

What's the reason for using hsl for everything?